src/main/java/fshop/db/FoodDaoImpl.java

```diff
@@ -0,0 +1,68 @@
+package fshop.db;
+
+import fshop.model.Food;
+import fshop.service.FoodService;
+import fshop.service.workfile.ReadCsv;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class FoodDaoImpl implements FoodDao {
+    private static final String BALANCE = "b";
+    private static final String COMA = ",";
+    private Map<Food, Integer> dataOfBalance;
+    private ReadCsv readCsv;
+    private FoodService foodService;
```

Comment:
Try to remove these dependencies from this class. Let's pass the payload information via methods

Reply:
so use these services in main method and pass the results of their job to the method of this service

Reply:
I can't remove this `private FoodService foodService;`, because I break logic into the method `void updateAll(List<String> fileLines)`. But readCsv I removed